examples/go-client/access.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"context"
+	"log"
+
+	"github.com/gravitational/teleport/lib/auth"
+	"github.com/gravitational/teleport/lib/services"
+)
+
+// accessRequests performs the necessary access management functions as an example
+func accessRequests(ctx context.Context, client *auth.Client) {
+	// create access request for api-admin to use the admin role in the cluster
+	accessReq, err := services.NewAccessRequest("api-admin", "admin")
+	if err != nil {
+		log.Printf("Failed to make new access request: %v", err)
```

Comment:
These log calls need to be followed by a `return` or this should use the API that os.Exits implicitly or this function should propagate the error to the caller.
Running as-is might result in panics.